website/static/js/formViewModel.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Maintains the base class for knockoutJS form ViewModels
+ */
+'use strict';
+
+var ko = require('knockout');
+
+var $osf = require('osfHelpers');
+
+
+var ValidationError = function(errorHeader, errorMessage) {
+    this.errorHeader = errorHeader;
+    this.errorMessage = errorMessage;
+    this.errorLevel = 'warning';
+};
+
+var FormViewModel = function() {
+
+    // Abstract method each ViewModel must impliment. Expected behavior is to either return true
+    // or an array of ValidationError objects
+    self.isValid = function() {
+        throw "Not Implemented";
```

Comment:
Always throw an `Error` object rather than a string.

```js
throw new Error('FormViewModel subclass must implement isValid');
```

Also, use single-quotes for strings: https://github.com/CenterForOpenScience/osf.io/blob/cd47834f67b2b9aefe6b7d41c433150a2c932e5f/.jshintrc#L11